Overall review of SDK docs 4.0.0 - closes #760 (#761)
LiskHQ/lisk-docs: modules/ROOT/pages/references/lisk-framework/index.adoc

==== subscribe

> This is used to subscribe to events occurring on the controller as shown below:

Reviewer: Here and in the lines below, there are still a lot of ` as shown below` statements. Please remove them all.
```suggestion
How to subscribe to events occurring on the controller:
```
